### PR TITLE
Create journal-of-clinical-neurophysiology.csl

### PR DESCRIPTION
**Pull request by damnation333**
via https://forums.zotero.org/discussion/comment/229467#Comment_229467

(based on a style ready under review (https://github.com/citation-style-language/styles/pull/2766), hence it's not validating yet)

### CODE REVIEW

File: journal-of-clinical-neurophysiology.csl [hunk 1]
```diff
@@ -13,8 +13,8 @@
     </author>
     <category citation-format="numeric"/>
     <category field="biology"/>
-    <issn>0736-0258</issn>
     <eissn>1537-1603</eissn>
+    <issn>0736-0258</issn>
```
Comment (rmzelle) [inline journal-of-clinical-neurophysiology.csl]: @damnation333, any reason you did this? I had already restarted the failing Travis build.

File: journal-of-clinical-neurophysiology.csl [hunk 2]
```diff
@@ -6,21 +6,21 @@
...
-    <eissn>1537-1603</eissn>
     <issn>0736-0258</issn>
+    <eissn>1537-1603</eissn>
     <updated>2017-06-25T12:04:18+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <macro name="author">
     <names variable="author" suffix=". ">
-      <name et-al-min="19" et-al-use-first="19" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+      <name et-al-min="7" et-al-use-first="3" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
```
Comment (rmzelle) [inline journal-of-clinical-neurophysiology.csl]: @adam3smith, I thought we had automatic checks against invalid et-al settings, but this didn't trigger any Travis errors. I'm not misremembering, right? Looks like I should investigate.

Comment (adam3smith) [inline journal-of-clinical-neurophysiology.csl]: I'm pretty sure you *are* misremembering this. We have talked about this, but I'm pretty sure we still have a pretty large number of styles with such settings.

Comment (rmzelle) [inline journal-of-clinical-neurophysiology.csl]: > I'm pretty sure you are misremembering this. We have talked about this, but I'm pretty sure we still have a pretty large number of styles with such settings.

See https://github.com/citation-style-language/styles/pull/2207.

Comment (adam3smith) [inline journal-of-clinical-neurophysiology.csl]: ah sorry,  you're right, I was wrong -- but the test checks for "greater or equal" when it should check for "greater", right?

Comment (rmzelle) [inline journal-of-clinical-neurophysiology.csl]: Ah, yes, good catch! (we should fix this in a PR so we can fix any problematic styles before Travis starts to complain; I'll put it on my todo list)

Comment (rmzelle) [inline journal-of-clinical-neurophysiology.csl]: (continued in https://github.com/citation-style-language/styles/pull/2777)

**Comment by rmzelle** [comment]
@damnation333, thanks! (you were sloppy on the et-al settings, though 😬; the instructions say `Provide all authors' names when fewer than seven; when seven or more, list the first three and add "et al."`)

**Comment by damnation333** [comment]
Yup. Sorry about that. I do indeed need to be careful. I've been pushing these out like a machine lately. Thanks for looking over them.
you also prefer eissn and then issn in that order, right?

**Comment by rmzelle** [comment]
> I do indeed need to be careful. I've been pushing these out like a machine lately. Thanks for looking over them.

Just so you know, I usually only look carefully at the style metadata (title, documention link, ISSNs, etc.) and take a very brief look at the style code. I don't have the time to actually test and check the output of styles, so we rely mostly on you for that.

> you also prefer eissn and then issn in that order, right?

No, first `<issn/>` and then `<eissn/>`. When in doubt, just run your style through http://formatter.citationstyles.org/, or look at existing styles in the repository, which should both give you the desired order.

**Pull request merged**